command_mode.c

```diff
@@ -1352,6 +1352,26 @@ static void cmd_pwd(char *arg)
 	}
 }
 
+static void cmd_rand(char *arg)
+{
+	editable_lock();
+	switch (cur_view) {
+	case TREE_VIEW:
+		break;
+	case SORTED_VIEW:
+		editable_rand(&lib_editable);
+		break;
+	case PLAYLIST_VIEW:
+		editable_rand(&pl_editable);
+		break;
+	case QUEUE_VIEW:
+		editable_rand(&pq_editable);
+		break;
+	}
+	editable_unlock();
+}
+
+
```

Comment:
Please keep whitespace between functions consistent with status quo in files you change, which is 1 empty line, not 2.